batch/Dockerfile.worker

```diff
@@ -18,21 +18,16 @@ RUN apt-get update && \
   rm -rf /var/lib/apt/lists/*
 
 COPY docker/requirements.txt .
-# re: pip install --upgrade pip: https://github.com/grpc/grpc/issues/22815
-RUN python3 -m pip install --upgrade pip && \
-    python3 -m pip install --no-cache-dir -U -r requirements.txt
+RUN hail-pip-install -r requirements.txt
```

Comment:
These are all going to fail now I think.

Reply:
I copied the script in above